src/ngx_postgres_rewrite.c

```diff
@@ -187,7 +187,7 @@
         url_variable.len = 0;
         //fprintf(stdout, "something here %s\n", p);
 
-        while(url_variable.len < (redirect + size) - (p + 1)) {
+        while((int)url_variable.len < (redirect + size) - (p + 1)) {
```

Comment:
@fridjon but len is length (size_t, really) and it can't be negative. I think it will be more sane to cast right part to (size_t).